keras_eval/eval.py

```diff
@@ -337,14 +339,39 @@ def _predict_folder(self, folder_path, verbose):
             # Predict
             if verbose:
                 print('Making predictions from model ', str(i))
-            probabilities.append(model.predict(images, batch_size=self.batch_size, verbose=1))
+            probabilities.append(model.predict(images, batch_size=self.batch_size, verbose=verbose))
 
         self.probabilities = np.array(probabilities)
         self.combined_probabilities = utils.combine_probabilities(self.probabilities, self.combination_mode)
         self.image_paths = image_paths
 
         return self.probabilities
 
+    def _predict_list(self, image_list, verbose=True):
+        '''
+
+        Predict the class probabilities of a set of images from a list.
+
+        Args:
+            image_list: List of images
+
+        Returns: Probabilities predicted, image path for every image (aligned with probability)
+
+        '''
+        probabilities = []
+        image_paths = []
+
+        for image_path in image_list:
+            # Read images from folder
+            if image_path.endswith(".png") or image_path.endswith(".jpeg") or image_path.endswith(".jpg"):
+                probabilities.append(self._predict_image(image_path, verbose)[0])
+                image_paths.append(image_path)
+
+        self.probabilities = np.array(probabilities)
+        self.image_paths = image_paths
+
+        return self.probabilities
```

Comment:
Are you saying in the function description that it returns the `image_paths` as well?

Reply:
It was just copied from the predict function, which was wrong too, changed it